backend/src/migrations/004_comment_response_refactor.ts

```diff
@@ -0,0 +1,73 @@
+import AccessRequestModel from '../models/AccessRequest.js'
+import ReleaseModel from '../models/Release.js'
+import ResponseModel, { ResponseKind } from '../models/Response.js'
+import ReviewModel from '../models/Review.js'
+import { toEntity } from '../utils/entity.js'
+
+export async function up() {
+  // Resolve release comments
+  const releases: any = await ReleaseModel.find({})
+  for (const release of releases) {
+    release.commentIds = []
+    if (release.get('comments') !== undefined) {
+      for (const releaseComment of release.get('comments')) {
+        const newComment = new ResponseModel({
+          user: toEntity('user', releaseComment.user),
+          kind: ResponseKind.Comment,
+          comment: releaseComment.message,
+          createdAt: releaseComment.createdAt,
```

Comment:
You should also likely add `updatedAt`, which should be set to the `createdAt` value.